host/src/containers/GameSessionContainer.tsx

```diff
@@ -187,33 +232,40 @@ const GameSessionContainer = () => {
   const handleBeginQuestion = () => {
     // I'm keeping this console.log in until we figure out NOT_STARTED so we can tell there's been a change in state 
     console.log(gameSession.currentState);
+    const gameSessionId = gameSession?.id;
+    const questionId = gameSession.questions[gameSession.currentQuestionIndex].id;
+    const order = gameSession.questions[gameSession.currentQuestionIndex].order;
```

Comment:
Personal opinion: Usually when accessing the same path multiple times, e.i. `gameSession.questions[gameSession.currentQuestionIndex]`, better to extract it into a variable and then use that variable:

```typescript
    const currentQuestion = gameSession.questions[gameSession.currentQuestionIndex];
    const questionId = currentQuestion.id;
    const order = currentQuestion.order;
```

Reply:
way cleaner, fixed